paper-tabs.html

```diff
@@ -376,6 +378,28 @@
         }, 10);
       },
 
+      _isItem: function(node) {
+        return this.items.indexOf(node) >= 0;
+      },
+
+      _onIronItemsChanged: function(event) {
+        this._onTabSizingChanged();
+
+        var _bindActivateHandler = this._bindActivateHandler;
+
+        event.detail.addedNodes
```

Comment:
`addedNodes` and `removedNodes` are actually NodeLists and not Arrays when returned by a MutationObserver, should I `Array.from` / `slice` this? Or is this like `Polymer.dom(...).querySelectorAll`, where its safe to assume an Array?